cmd/validator/accounts/wallet_utils.go

```diff
@@ -30,7 +30,10 @@ func walletWithKeymanager(c *cli.Context) (*wallet.Wallet, keymanager.IKeymanage
 }
 
 func walletWithWeb3SignerKeymanager(c *cli.Context, config *remote_web3signer.SetupConfig) (*wallet.Wallet, keymanager.IKeymanager, error) {
-	w := wallet.NewWalletForWeb3Signer()
+	w, err := wallet.NewWalletForWeb3Signer(c)
+	if err != nil {
+		return nil, nil, errors.Wrap(err, "unable to use wallet directory for remote signer")
```

Comment:
I think this error should be more generic since it doesn't really map to the explanation of `NewWalletForWeb3Signer`
```suggestion
		return nil, nil, errors.Wrap(err, "unable to create new wallet for remote signer")
```
Then have the dir error in the function reflect which part failed.